exo/main.py

```diff
@@ -221,16 +267,36 @@ def handle_exit():
       print("Error: Model name is required when using 'run' command or --run-model")
       return
     await run_model_cli(node, inference_engine, model_name, args.prompt)
+  elif args.command == "eval" or args.command == 'train':
+    data_path = args.data
+    train, val, test = load_dataset(data_path)
+    model_name = args.model_name
+    if args.command == 'eval':
+      if not model_name:
+        print("Error: Much like a human, I can't evaluate anything without a model")
+        return
+      await eval_model_cli(node, inference_engine, model_name, test, args.batch_size)
+    else:
+      if not model_name:
+        print("Error: This train ain't leaving the station without a model")
```

Comment:
again, an actionable command would be helpful here (at the end! I'm not saying don't keep this part of the message 😄 )